python/cuml/cuml/experimental/fil/fil.pyx

```diff
@@ -608,6 +609,13 @@ class ForestInference(UniversalBase, CMajorInputTagMixin):
             handle=handle, verbose=verbose, output_type=output_type
         )
 
+        if output_class is not None:
+            warnings.warn(
+                "Parameter `output_class` was deprecated in version 25.06 and will be "
+                "removed in 25.08. Use `is_classifier` instead.",
```

Comment:
Are those two parameters essentially equivalent? Can we still respect `output_class` as part of the deprecation?

Reply:
Yes, `is_classifier` is equivalent to `output_class`.

> Can we still respect `output_class` as part of the deprecation?

`output_class` is already respected (see line 624), but let me move the line close to the warning so that it's easy to find. I will also update the warning message to indicate that `output_class` is respected.